.github/workflows/continuous-integration.yml

```diff
@@ -21,28 +24,32 @@ jobs:
         ports:
           - 5432:5432
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm install
-      - run: npm test
+      - name: Update NPM
+        if: matrix.node-version <= 14
+        run: npm install -g npm@latest-7
+      - name: Install dependencies
+        run: npm ci
```

Comment:
if `--ignore-scripts` works, we should keep it there. This can be used to signal that this package works without the `postinstall` scripts executing (i.e. for the more security-conscious users).